pybleau/app/plotting/plot_style.py

```diff
@@ -114,16 +115,13 @@ class BaseXYPlotStyle(HasStrictTraits):
     range_transform = Callable
 
     def __init__(self, **traits):
+        super(BaseXYPlotStyle, self).__init__(**traits)
+        self.renderer_style_manager.view_klass = self.view_klass
+
         # Support passing the renderer_styles directly since the
         # renderer_style_manager is there just for UI layout:
-        renderer_styles = None
         if "renderer_styles" in traits:
             renderer_styles = traits.pop("renderer_styles")
```

Comment:
What's the point of working on the traits dict after super has been called?

Reply:
Yes! Thanks for catching this. I think this was a change that didn't get completely reverted back when I was trying out something with the `renderer_styles`. Will revert.

> What's the point of working on the traits dict after super has been called?